x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/chart_switch.tsx

```diff
@@ -294,24 +296,46 @@ export const ChartSwitch = memo(function ChartSwitch(props: Props) {
                     prepend: (
                       <EuiIcon className="lnsChartSwitch__chartIcon" type={v.icon || 'empty'} />
                     ),
-                    append:
-                      v.selection.dataLoss !== 'nothing' ? (
-                        <EuiIconTip
-                          aria-label={i18n.translate('xpack.lens.chartSwitch.dataLossLabel', {
-                            defaultMessage: 'Warning',
-                          })}
-                          type="alert"
-                          color="warning"
-                          content={i18n.translate('xpack.lens.chartSwitch.dataLossDescription', {
-                            defaultMessage:
-                              'Selecting this chart type will result in a partial loss of currently applied configuration selections.',
-                          })}
-                          iconProps={{
-                            className: 'lnsChartSwitch__chartIcon',
-                            'data-test-subj': `lnsChartSwitchPopoverAlert_${v.id}`,
-                          }}
-                        />
-                      ) : null,
+                    append: (
+                      <EuiFlexGroup
+                        gutterSize="xs"
+                        responsive={false}
+                        style={{ display: 'inline-flex' }}
```

Comment:
Minor nitpick, but since we already have the SCSS file, would it be better to just apply a class to this `EuiFlexGroup` and put the `display: inline-flex;` style in the `chart_switch.scss` file? That way all the styles are in one place.

Reply:
Added CSS class in https://github.com/elastic/kibana/pull/97978/commits/6d35fb72f453d08422c6e27b43236816d91ba0ee